lister/test/lister_test.exs

```diff
@@ -0,0 +1,128 @@
+defmodule ListerTest do
+  use ExUnit.Case
+  doctest Lister
+
+  @test_count 100
+
+  describe "should implement Listable protocol for String" do
+    test_cases =
+      StreamData.string(:alphanumeric)
```

Comment:
I absolutely love using property-based tests here, but what’s wrong with [`use ExUnitProperties`](https://hexdocs.pm/stream_data/ExUnitProperties.html#__using__/1) and get all the goodness from it, like `check all` etc?

Also, does not `@tag :skip` skip the test?

Reply:
`@tag :skip` already removed :) I will try to rewrite this to `use ExUnitProperties`

Reply:
Tests had been refactored using `check all` from `ExUnitProperties`